lib/widgets/message_list.dart

```diff
@@ -892,6 +893,16 @@ class MessageWithPossibleSender extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final message = item.message;
+    bool messageEdited = false, messageMoved = false;
+    if(message.editHistory != null) {
+      if((message.editHistory?[message.editHistory!.length-1].topic !=
+          message.editHistory?[message.editHistory!.length-1].prevTopic) ||
```

Comment:
This is pretty messy to read. The repeated subexpression here should be pulled out as a local variable.

Also see the `Iterable.last` getter :-)